libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -1942,3 +1950,9 @@ bool AP_GPS_UBLOX::supports_F9_config(void) const
 {
     return _hardware_generation == UBLOX_F9 && _hardware_generation != UBLOX_UNKNOWN_HARDWARE_GENERATION;
 }
+
+// return true if GPS is an uBlox M9
+bool AP_GPS_UBLOX::is_ublox_M9(void) const
```

Comment:
We should be checking for capability/requirements rather than device model.

Reply:
Agreed, this helper should go away.